### PR TITLE
Tiny fixes found while preparing next session of the advanced course

### PR DESCRIPTION
**Pull request by sponce**

### CODE REVIEW

File: talk/basicconcepts/assert.tex [hunk 1]
```diff
@@ -77,7 +77,7 @@
     \begin{cppcode*}{}
       double f(UserType a) {
         static_assert(
-          std::is_floating_point<UserType>::value,
+          std::is_floating_point_v<UserType>,
```
Comment (bernhardmgruber) [inline talk/basicconcepts/assert.tex:80]: Requires C++17 now, but the slide says 11. I guess this was the reason, the former version was used. But I totally agree that we should teach the students to use the `_v` traits!

**Review by bernhardmgruber — APPROVED** [review]
Small nit, otherwise good!

File: talk/basicconcepts/assert.tex (hunk 1 above)
Comment (sponce) [reply talk/basicconcepts/assert.tex]: Well spotted ! I added a comment "C++17" on the given line, leaving 11 at the top as it targets the static_assert there.

Comment (bernhardmgruber) [reply talk/basicconcepts/assert.tex]: Parfait!

**Review by bernhardmgruber — APPROVED** [review]

**Pull request merged**